### DISCUSSION TITLE
SWAR plots stuck at 100.0%?

### DISCUSSION THREAD

**Discussion by thust777** [swar/Swar-Chia-Plot-Manager#492] [Q&A]
Hey folks,

As seen below, my plots are getting stuck at 100.0%. I'm on Windows 10, using Sabrent 3s. 3900x. 32GB RAM. Transferring Plots to a 12TB Mybook, which Task Manager shows as performing at 100% Active Time.

Any ideas re how to fix this? Thanks yall.

![swar](https://user-images.githubusercontent.com/84170080/118524385-9ec42c80-b72d-11eb-9a4f-44a6bfea47a1.jpg)

**Comment by infpico** [comment]
probably its still running/writing to your hdd, even at 100%

**Comment by farkeytron** [comment]
Same thing happens to me... it's just moving the finished plots... sometimes it takes a while.

**Comment by lipefur** [comment]
<img width="1065" alt="Captura de Tela 2021-05-18 às 09 44 10" src="https://user-images.githubusercontent.com/51271654/118653347-cebb1080-b7bd-11eb-9e76-b996746ccf86.png">

I also have the same problem
is not sending to the final disk

**LOG**
```Time for phase 4 = 1581.322 seconds. CPU (29.920%) Tue May 18 06:35:14 2021
Approximate working space used (without final file): 269.363 GiB
Final File size: 101.366 GiB
Total time = 21343.409 seconds. CPU (106.980%) Tue May 18 06:35:14 2021
Could not copy "A:\\temp1\\plot-k32-2021-05-18-00-39-2a399df45c34543f8da6d0e41705d0c3414b92867796feff0ec45dac566ab9c7.plot.2.tmp" to "F:\\plots\\plot-k32-2021-05-18-00-39-2a399df45c34543f8da6d0e41705d0c3414b92867796feff0ec45dac566ab9c7.plot.2.tmp". Error Parâmetro incorreto.. Retrying in five minutes.
```

**Config.yaml**
```
temporary_directory: A:\temp1
destination_directory: F:\plots
```

**Reply by laroy-sh** [reply]
Yeah, I've got the same issue
![Screenshot 2021-05-18 224129](https://user-images.githubusercontent.com/25833114/118720522-fb0c7680-b829-11eb-8b06-ed709ed94a59.png)

**Reply by lipefur** [reply]
I managed to solve the problem
The final directory was formatted in MSDOS, when I formatted in NTFS the problem was solved.


note: I use windows

**Comment by xnotx2** [comment]
If you're trying to transfer it to a network drive you need to give full permissions. Once it's full you can put it back to read only.
You have to go into the computer that is sharing the drive and change the sharing permissions of the drive by right clicking on it, selecting sharing tab, then clicked advanced sharing, then click permissions and tick everyone.

**Reply by thust777** [reply]
Interesting. I've just applied that setting. I'll let you folks know if it fixes the problem.

**Reply by xnotx2** [reply]
I should clarify this part "you need to give full permissions. Once it's full you can put it back to read only."
I mean give the shared drive full control, then when the drive has been filled with plots you can change it back to read only.
